Example/PrimerSDK/MerchantCheckoutViewController.swift

```diff
@@ -86,17 +85,8 @@ class MerchantCheckoutViewController: UIViewController {
         )
 
         
-        Primer.shared.configure(settings: generalSettings)
-        Primer.shared.configure(theme: CheckoutTheme.primer)
+        Primer.shared.configure(settings: generalSettings, theme: CheckoutTheme.primer)
         Primer.shared.delegate = self
```

Comment:
I'm quite sure I've added it as part of the [Notion suggestion](https://www.notion.so/Checkout-Components-API-b917503bb5ca4b7da3d2af031e8208f9?d=b9fa7eb6a59243199b0db17a928f7bfc#5b655d7552354f9d92956973fb02eb60) too - was wondering if we'd add the `delegate` as part of the configuration func (with default value to `nil` perhaps)